custom_components/places/sensor.py

```
@@ -429,25 +460,51 @@ def import_attributes_from_json(self, json_attr: MutableMapping[str, Any]) -> No
+        Args:
+            attr: Attribute key to inspect.
+
+        Returns:
+            ``True`` when the value is missing or falsey, with numeric zero
+            treated as a meaningful value.
+        """
         if self._internal_attr.get(attr) or self._internal_attr.get(attr) == 0:
             return False
         return True
 
     def get_attr(self, attr: str | None, default: Any | None = None) -> None | Any:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `default`

(ANN401)